msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -1157,6 +1158,21 @@ The `extensions.runtimes` array configures the sets of runtimes and actions that
 
 To use `extensions.runtimes`, see [create add-in commands](/office/dev/add-ins/develop/create-addin-commands-unified-manifest), [configure the runtime for a task pane](/office/dev/add-ins/develop/create-addin-commands-unified-manifest#configure-the-runtime-for-the-task-pane-command), and [configure the runtime for the function command](/office/dev/add-ins/develop/create-addin-commands-unified-manifest#configure-the-runtime-for-the-function-command).
 
+### extensions.contentRuntimes
+
+The `extensions.contentRuntimes` array configures pages of content that are embedded in an Excel or PowerPoint document. 
+
+|Name| Type| Maximum size | Required | Description|
+|---|---|---|---|---|
+|`id`| String | 64 characters | ✔️ | Specifies the ID for runtime. Must be unique across all `runtimes` and `contentRuntimes` objects. |
+|`code`| Object | | ✔️ | Specifies the location of content for the runtime. |
+|`code.page`| String | 2048 charaters| ✔️ | The full URL of the page that contains the content that is embedded in the document. |
+|`code.script`| String | | | Don't use this property under `extensions.contentRuntimes`.|
```

Comment:
The "Don't use this property" description doesn't really provide a reason not to use it. How about something like:

Not applicable for `extensions.contentRuntimes`.

Reply:
Hi @Rick-Kirkham,
I like the suggestion made by Erika and incline to change the sentence as suggested. What are your thoughts?